VideoLocker/src/main/java/org/edx/mobile/module/analytics/ISegmentImpl.java

```diff
@@ -594,15 +597,62 @@ public Properties trackUserDoesNotHaveAccount() {
 
     }
 
+    /**
+     * This function is used to track if user clicks on Find Courses
+     * @return
+     */
     @Override
     public Properties trackUserFindsCourses() {
         try{
-            VideoEvent vEvent = new VideoEvent();
-            vEvent.properties.putValue(Keys.NAME, Values.USER_FIND_COURSES);
-            vEvent.setAppNameContext();
+            SegmentAnalyticsEvent aEvent = new SegmentAnalyticsEvent();
+            aEvent.properties.putValue(Keys.NAME, Values.USER_FIND_COURSES);
+            aEvent.setAppNameContext();
 
-            tracker.track(Keys.FIND_COURSES, vEvent.properties);
-            return vEvent.properties;
+            tracker.track(Keys.FIND_COURSES, aEvent.properties);
+            return aEvent.properties;
+        }catch(Exception e){
+            logger.error(e);
+        }
+        return null;
+    }
+
+    /**
+     * This function is used to track if user clicks on Create Account on registration screen
+     * @return
+     */
+    @Override
+    public Properties trackCreateAccountClicked() {
+        try{
+            SegmentAnalyticsEvent aEvent = new SegmentAnalyticsEvent();
+            aEvent.properties.putValue(Keys.NAME, Values.CREATE_ACCOUNT_CLICK);
+            aEvent.setAppNameContext();
```

Comment:
Let's hear back from the Analytics team.  They may want to track additional fields in this event.  But for now, you can go ahead with this.